website/docs/advanced/strategy-constraints.md

```diff
@@ -5,52 +5,174 @@ title: Strategy Constraints
 
 :::info Availability
 Strategy constraints are available to Unleash Pro and Enterprise users.
+
+Unleash 4.9 introduced a more comprehensive set of constraint operators. These require that both Unleash *and* your client SDK of choice support them. See the [SDK compatibility table](../sdks/index.md#server-side-compatibility-table) for more information.
```

Comment:
This may be a separate conversation, but we don't mention _which_ version this is available from for the SDKs.

Reply:
Indeed we don't! As far as I know, the SDKs don't align on versions, so it won't be the same across the board. Instead, they'll have to check their SDKs specifically. 

It would be nice if that could be worked into the compat table somehow, but it would take some effort. I also like the simple visual distinction that you get with the emoji, so I'm not sure we should just change it to be numbers. Got any ideas how we could surface this?